src/mongodb/mongodb_models/bills_model.py

```diff
@@ -1,6 +1,11 @@
 from mongoengine import Document, StringField, FloatField
 
+from src.env_utils import MongoDBSettings
 
-class Bill(Document):
+
+class BillDocument(Document):
```

Comment:
I don't understand this type.
If it is the amount a patron owes, why can't we just have this as a field on patrons?

Reply:
Fran told me that he want that all the bills and the economic management of the library will be separate department.